script.js

```diff
@@ -231,4 +231,25 @@ closeMobilePopUp.forEach((button) => {
     const modal = button.closest('.mobile-popup-model');
     closeModal(modal);
   });
-});
\ No newline at end of file
+});
+
+// Contact form validation goes there
+
+// const contactForm = document.querySelector('.contactForm');
+// const emailInput = document.querySelector('#lowerCase');
+// const isValid = (emailInput) => !/([A-Z])/g.test(emailInput);
+// const error = document.querySelector('.error');
+
+// contactForm.addEventListener ('submit', (event) => {
+//   if (!isValid(emailInput.value)){
+//     event.preventDefault();
+//     error.textContent = 'Email must be in lower case, darling!';
+//     error.className = 'error active';
+//   }else {
+//     form.submit();
+//     emailInput.className= 'valid';
+//     error.textContent = '';
+//     error.className = 'error';
+//     save();
+//   }
+// })
```

Comment:
- [ ] Although you have performed excellently and the necessary features are functioning correctly :+1:, it is necessary to adhere to the best practices in JavaScript. Therefore, kindly remove the commented-out code. This is important because commented-out code may cause confusion, reduce the code's readability, and make it difficult to maintain in the future. Best practices in coding emphasize the importance of keeping code clean, organized, and easy to read. Removing unnecessary code, such as commented-out code, helps to achieve these goals.